### PR TITLE
Make the mocks faster

### PR DESCRIPTION
**Pull request by jiancheung**
Direct manipulation of mock's clever.db might also need to update clever.map

### CODE REVIEW

**Comment by aashay** [comment]
This LGTM but I'm the absolute worst at coffeescript, so consider me generally useless.

File: mock.coffee [hunk 1]
```diff
@@ -32,6 +33,24 @@ module.exports = (api_key, data_dir) ->
     schools: trequire("#{data_dir}/schools")
     schoolproperties: trequire("#{data_dir}/schoolproperties")
 
+  # clever.map[resources] are maps from ids to resources (for faster lookup)
+  # This doesn't just replace clever.db for compatibility reasons..for now..
```
Comment (kshen0) [inline mock.coffee]: Mention of private repo will be irrelevant to public consumers

**Comment by kshen0** [comment]
Can you document (in comments in `mock.coffee`)
* the current way to perform mock writes to the Clever object
* that the mock interface is still in beta and subject to change

**Comment by kshen0** [comment]
Looks good

**Pull request merged**